api/terraform/modules/catalogue_api_stack/domains.tf

```diff
@@ -1,27 +1,35 @@
-data "aws_route53_zone" "dotorg" {
-  provider = "aws.routemaster"
+/*data "aws_route53_zone" "dotorg" {
```

Comment:
What's going on here?

Reply:
We can’t (currently) look up the hosted zone ID programmatically in tf 0.12. I’ve got a patch out to give the role the right permissions, so I’m hoping this will be back to the uncommented Code extremely soon.